java/registry/src/main/resources/public/_schemas/UserFieldStatus.json

```diff
@@ -66,6 +66,8 @@
 
     "privateFields": [],
     "signedFields": [],
+    "indexFields": ["loginId"],
+    "uniqueIndexFields": ["loginId"],
     "systemFields": ["createdOn", "updatedAt", "createdBy", "updatedBy"]
```

Comment:
use same semantic :  some place we have createdAt and here we have createdOn.

Reply:
done.